src/System.Management.Automation/engine/CommandCompletion/PseudoParameterBinder.cs

```diff
@@ -1338,79 +1335,76 @@ private bool PrepareCommandElements(ExecutionContext context)
             if (_commandAst.IsInWorkflow())
             {
                 var converterType = Type.GetType(Utils.WorkflowType);
-                if (converterType != null)
+                var activityParameters = (Dictionary<string, Type>) converterType?.GetMethod("GetActivityParameters").Invoke(null, new object[] { _commandAst });
+                if (activityParameters != null)
```

Comment:
In the future, I'd prefer to see language usage changes like this as a single PR, and I'd prefer to see 1 commit per language usage change.